content/go/concepts/strings/terms/trim/trim.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Trim()'
+Description: 'Removes leading and trailing specified characters from a string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Trim()`** function removes leading and trailing specified characters from a string. The `strings` library must be imported in order to use this function.
+
+## Syntax
+
+```pseudo
+trimmed := strings.Trim(str, cutset)
+```
+
+Where `str` is the input string from which the leading and trailing characters are to be removed, `cutset` is a string containing the characters to be removed, and `trimmed` is the resulting string after trimming.
```

Comment:
I would suggest reformatting this as a list-
- `str`: The input string from which the leading and trailing characters are to be removed.
- etc